### ISSUE TITLE
Can't get it work on Windows

### ISSUE THREAD

**Issue by BrownBichon** [seblavoie/After-Effects-Scripting-Sublime-Text-Package#9]
**I'm using Sublime Text 3 portable version and After Effects CC 2015 on Windows 10.**

By default, I got this weird error message:
![cap](https://cloud.githubusercontent.com/assets/10085220/15831499/8375445e-2c50-11e6-90b9-f6c327896b36.JPG)

Then I wrote a `test.jsx` on my desktop and modified the `build.bat` as below:
```shell
@echo off
:: Renaming arguments
set file_name=test.jsx
set file_path=D:\Desktop

:: Change this accordingly to your CS version
set version=CC 2015

:: Adobe After Effects folder location
set base_path=c:\Program Files\Adobe
set ae_folder_path=%base_path%\Adobe After Effects %version%
set ae_scripts_folder_path=%ae_folder_path%\Support Files\Scripts


cd "%file_path%"

:: Copying script to Scripts folder
copy "%file_name%" "%ae_scripts_folder_path%\%file_name%"

cd "%ae_folder_path%\Support Files"

:: Running script in After Effect
AfterFX -r "%ae_scripts_folder_path%\%file_name%"

:: Printing happy feedback in the console
echo "Successfully compiled %file_name% to %ae_scripts_folder_path%\%file_name%";

pause
```

- I double-click the `build.bat`, no error but the script didn't execute.
- I ran the `build.bat` as Administer, the script executed.

But I still can't figure out how to get it work inside Sublime.
Maybe the `"encoding": "cp850"` in the `AfterEffectsScripting.sublime-build` is the issue, or the writing access of the script folder is the issue.
I tried removing the `"encoding": "cp850"` line, the first error line disappear, but still no working.

I got this package running perfectly on my Mac, really helpful. Hope you can solve this issue.

Cheers!

**Comment by seblavoie** [comment]
It has been a while since I've had access to a windows environment. I will unfortunately not be able to help you out with this one. Although let me know if you find the solution and I'd be willing to apply it to the repo.

**Comment by BrownBichon** [comment]
Hey, Since I am working on Windows with Chinese. I changed the `"encoding": "cp850"` to `"encoding": "cp936"`, the messy code turns to be `The filename, directory name, or volume label syntax is incorrect`. But since I'm really a beginner of coding, I don't know how to slove it.

But as I mentioned in the last comment, the modified `build.bat` is OK to use as Administer on Windows 10. Today I tested it on Windows 7, it works when just double-clicking. So I assume the line `"cmd": ["cd ${packages}\\AfterEffects\\ && build.bat $file_name $file_path"],` in `AfterEffectsScripting.sublime-build`  causes the issue. Maybe `build.bat` is not getting the correct `file_name` and `file_path`.

I'll keep learning and trying, hope I can get the package work.
